_posts/2016-10-15-recurse-center-week-3.md

```diff
@@ -0,0 +1,132 @@
+
+I started wondering if my offers had value. Or whether the quality of these interactions were as good as I thought.
+
+#### **I wondered if RC was wrong for me.**
+
+Another thought was that maybe RC just wasn't compatible with me.
+I actually got rejected the first time I applied. Not even after an interview...right after the application.
+
+Maybe my social goals weren't aligned with this space.
+Maybe RC is more focused on the technical growth.
+Maybe I didn't belong.
+
+#### **I focused on things outside the space.**
+
+I realized there was a lot for me to explore outside of RC. As a result, no evenings were spent 100% at RC last week.
+
+I went to shows, tech events and meet-ups. This relieved a lot of the pressure I was putting on RC. Instead of expecting RC to fulfill all my needs, I was being proactive in seeking out alternative spaces. It helped me disassociate a bit and regain some sense of autonomy.
+
+#### **I connected with other RCers.**
+
+Every event I went to ended up having fellow RCers. Being in smaller groups outside the space allowed us to talk a bit more about our lives outside of RC. I heard a few others were having similar troubles feeling connected at RC.
+
+Learning that helped me once again feel connected to the space. Things improved with each day and by the end of the week I felt a lot better about everything. ☺️
+
+### Social 💃
+
+This week was super social! Was able to go to an event every night except Wednesday. Half of the events were things I planned for myself and the other half things I learned about through RC. None of them were in the RC space.
+
+#### **I tend to socialize predominantly outside of work.**
+
+Something I was thinking about was how much more I'm used to socializing outside of my work place. I've always really enjoyed the people I work with, but naturally tend to find my best connections in external groups. Not sure what originally made me think fulfilling all my social needs at RC would work...
+
+### Intellectual 🤔
+
+#### **Experimental Checkins**
+
+This week I tried something called "Experimental Checkins".
+
+In regular checkins, you meet in the morning and share what you're working on. A few people found that wasn't as valuable as they would like and decided to do experimental checkins instead.
+
+The idea with experimental checkins is to share what you're working on, but additionally get help from other members of the group. It encourages collaboration and gets everyone more invested in the success of their peers.
+
+I found these checkins to be extremely valuable. I found myself more invested in helping others, and in turn felt supported by them.
+
+#### **Shorter Days**
+
+Unfortunately, these checkins are also scheduled a bit later in the day. I've been using checkins in to set the latest time I can arrive at RC. With normal checkins, that meant I was arriving at 10:30. With experimental checkins, I don't need to be in until 11:30.
```

Comment:
yes, i would definitely go to experimental checkins if they were at 10:30. since i don't want to stay in the space too late into the evening, it's good for me to have an incentive to arrive earlier